packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -62,8 +62,9 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
         SynchronousQueue()
     )
 
-    val logsPlugin: DatadogLogsPlugin = DatadogLogsPlugin()
-    val rumPlugin: DatadogRumPlugin = DatadogRumPlugin()
+    // Because Flutter recreates the plugin, we have to rely on singletons for the contained plugins
+    val logsPlugin: DatadogLogsPlugin = DatadogLogsPlugin.instance
+    val rumPlugin: DatadogRumPlugin = DatadogRumPlugin.getInstance()
```

Comment:
Yes, though it might not be a good reason? `DatadogRumPlugin` can take a parameter to its constructor which is used in unit tests, while `DatadogLogsPlugin` does not, so I took separate approaches to creating / getting the instances (`lazy` vs. `getInstance()`)

Do you think I should try to standardize them more?

Reply:
This is not a blocker for me definitely, I was just curious of the reason :) So as you wish.

Reply:
Nope, nevermind, I switched to using `by lazy` for both, since constructors need to be marked as internal anyway.